docker/include/bin/start_faux

```diff
@@ -156,11 +156,10 @@ elif [ -n "${options[bacnetfail]}" ]; then
          FauxDeviceEngine.EntryPoint $local_ip $broadcast_ip "Faux-Device-Fail.json"  &
 fi
 
-# NTPv4 query to the NTP server learnt from DHCP.
+# NTPv4 query to the local NTP server
 # NTPv3 query to the IP of time.google.com (since resolv.conf is modified by other tests)
 if [ -n "${options[ntpv4]}" ]; then
-    dhcp_ntp=$(fgrep NTPSERVERS= /run/ntpdate.dhcp)
-    ntp_server=`echo $dhcp_ntp | cut -d "'" -f 2`
+    ntp_server=ntp.daqlocal
```

Comment:
since ntp.daqlocal is used a couple of times, make it a named variable in this file? (LOCAL_NTP_SERVER_NAME=ntp.daqlocal or something like that)